packages/mui-base/src/SliderUnstyled/SliderUnstyledProps.ts

```diff
@@ -204,7 +205,7 @@ export interface SliderUnstyledTypeMap<P = {}, D extends React.ElementType = 'sp
  * Utility to create component types that inherit props from SliderUnstyled.
  */
 export interface ExtendSliderUnstyledTypeMap<M extends OverridableTypeMap> {
-  props: M['props'] & SliderUnstyledTypeMap['props'];
+  props: M['props'] & SliderUnstyledTypeMap['props'] & { component?: React.ElementType };
```

Comment:
cc @michaldudak you want to only leave these slider changes on this PR, so that you don't need to create a new one?

Reply:
Let's close it. I'll create another one to remove BackdropUnstyled altogether.